test/info/info-help.test.js

```diff
@@ -29,8 +29,8 @@ describe('should print help for info command', () => {
         expect(stderr).toHaveLength(0);
     });
 
-    it('should respect the --color=false flag', () => {
-        const { stdout, stderr } = runInfo(['info', 'help', '--color=false']);
+    it('should respect the color default true ', () => {
+        const { stdout, stderr } = runInfo(['info', 'help']);
```

Comment:
Do not change existing tests if it is unnecessary

Reply:
Okay.

Reply:
I am having trouble finding out `--color` flag on `v4`. It is in the WIP or something? That's why my test were failing locally. So, I have modified it little bit.